Parse/BuildParseTree.pas

```diff
@@ -2933,6 +2936,7 @@ procedure TBuildParseTree.CheckLabelPrefix;
   lbColonSecond: boolean;
 begin
   lc2 := fcTokenList.SolidToken(2);
+  if lc2=nil then exit;
```

Comment:
the same here: `lc2=nil`...
Also, "one statement for a line": `if statement` - one line, `Exit` is on the next...
```delphi
if lc2 = nil then
  Exit;
```

Reply:
ok